cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -364,13 +364,24 @@ struct HeffteBackendTraits<Kokkos::Cuda, Impl::FFTBackendDefault>
 };
 #endif
 #endif
+#ifdef Heffte_ENABLE_ROCM
 #ifdef KOKKOS_ENABLE_HIP
 template <>
 struct HeffteBackendTraits<Kokkos::Experimental::HIP, Impl::FFTBackendDefault>
 {
     using backend_type = heffte::backend::rocfft;
 };
 #endif
+#endif
+#ifdef Heffte_ENABLE_ONEAPI
```

Comment:
In previous versions of heFFTe, the backend type-tags were defined only if the corresponding preprocessor variable existed too.

In the latest master branch, all backend tags are always defined with the type-traits specialized structure `heffte::backend::is_enabled<type_tag>::value` to define whether it is available or not. This allows for a more C++-ish handing of these types.

At any rate, this is a F.Y.I.